squad/core/comparison.py

```diff
@@ -164,11 +164,16 @@ class TestComparison(BaseComparison):
 
     __test__ = False
 
-    def __init__(self, *builds):
+    def __init__(self, *builds, regressions_and_fixes_only=False):
         self.__intermittent__ = {}
+        self.regressions_and_fixes_only = regressions_and_fixes_only
         BaseComparison.__init__(self, *builds)
 
     def __extract_results__(self):
+        if self.regressions_and_fixes_only:
+            self.__regressions_and_fixes__(self.builds[0], self.builds[1])
+            return
+
```

Comment:
I don't see `regressions_and_fixes_only` being used anywhere. So `__regressions_and_fixes__` below is not being called anywhere.

Reply:
Ah, forgot to use that in `ProjectStatus.create_or_update`